projects/packages/my-jetpack/_inc/components/upsell-banner/index.jsx

```diff
@@ -0,0 +1,82 @@
+import { Gridicon, Button } from '@automattic/jetpack-components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import { CardWrapper } from '../card';
+
+import './style.scss';
+
+/**
+ * Upsell banner component.
+ *
+ * @param {object} props - Component props.
+ * @returns {object} UpsellBanner React component.
+ */
+const UpsellBanner = props => {
+	// todo: This component could be extracted into js-packages/components. And also the Jetpack Card component
+	const {
+		icon,
+		title,
+		description,
+		cta1Label,
+		cta1URL,
+		cta1Target,
+		cta1WithIcon,
+		cta1OnClick,
+		cta2Label,
+		cta2URL,
+		cta2Target,
+		cta2WithIcon,
+		cta2OnClick,
+	} = props;
+
+	return (
+		<CardWrapper className="upsell-banner">
+			{ icon && (
+				<div className="upsell-banner--icon">
+					<img
+						src={ icon }
+						alt={ __( 'A control panel representing Jetpack Manage', 'jetpack-my-jetpack' ) }
```

Comment:
What do you think about leaving this alt value blank, since the icon is decorative? That would also save us from saying "Jetpack Manage" in a shared component.

Reply:
Yep, it is okay to leave it blank, and it makes sense if it's a shared component. I borrowed it from another component in the Jetpack plugin.

Reply:
Updated here: https://github.com/Automattic/jetpack/pull/35078/commits/2683c910e08b252d089210d4c7019eeb028cc82c